lib/common_test/doc/src/ct_suite.xml

```diff
@@ -135,11 +138,12 @@
         <v><seetype marker="#ct_group_def">ct_group_def()</seetype> = {GroupName, Properties, GroupsAndTestCases}</v>
         <v>GroupName = <seetype marker="#ct_groupname">ct_groupname()</seetype></v>
         <v>Properties = [parallel | sequence | Shuffle | {RepeatType, N}]</v>
-        <v>GroupsAndTestCases = [Group | {group, GroupName} | TestCase]</v>
+        <v>GroupsAndTestCases = [Group | {group, GroupName} | TestCase | {testcase, TestCase, TestCaseRepeatType}]</v>
```

Comment:
(Ingela) Verify if testcase tuple can also contain 2 elements only and if this should be specced here ...
see 2 element testcase tuple https://www.erlang.org/doc/man/ct.html#run_test-1

Reply:
2 element testcase tuple does not seem to work in test suite code. Therefore, I will not add it.